Solution.py

```diff
@@ -0,0 +1,29 @@
+"""
+Time Complexity Log n 
+Space complexity O(1)
```

Comment:
Shouldn't the time be nlogn as you are sorting

Reply:
Yeah write I forgot the sorting part . Thanks for pointing it out